hack/setup_env.sh

```diff
@@ -24,7 +24,7 @@ function golang() {
   echo "Installing golang..."
   curl --retry 5 -L https://storage.googleapis.com/golang/getgo/installer_linux -o /tmp/golang_installer
   chmod u+x /tmp/golang_installer
-  /tmp/golang_installer
+  /tmp/golang_installer -version 1.18
```

Comment:
it's probably not critical that we were using golang 1.17 back then, right?

Reply:
that should be fine

Reply:
anyway, this branch will be deprecated soon, no?

Reply:
right, October 13